imgui/src/input_widget.rs

```diff
@@ -530,19 +533,36 @@ impl<'ui, 'p, L: AsRef<str>> InputFloat<'ui, 'p, L> {
             value,
             step: 0.0,
             step_fast: 0.0,
+            display_format: None,
```

Comment:
we should default this display format to `"%.3f"`, as it originally was. That also means that display_format can actually become `F` instead of `Option<F>`,  and start life with the type `&'static str`

Reply:
Just for the float case I presume? For `InputScalar` I'd definitely stick with letting imgui pick the format.